src/main/java/io/confluent/connect/jdbc/sink/SchemaMismatchException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.connect.jdbc.sink;
+
+import java.sql.SQLException;
+
+public class SchemaMismatchException extends SQLException {
```

Comment:
This design choice also forces you to re-wrap the exception later in order to pass through the `catch SQLException` statement.
It'd simplify the handling logic if we confidently knew that SQLExceptions were retriable, and this class/it's usages would break that assumption.

Reply:
Hmmm I see your point, WDYT of extending `ConnectException` instead then? Would probably have to change how we handle exception throwing/logging

Reply:
I think extending `ConnectException` makes more sense here, yeah.